src/Skipper/Matcher/FileInfoMatcher.php

```diff
@@ -37,8 +35,11 @@ public function doesFileInfoMatchPatterns(string $filePath, array $filePatterns)
     /**
      * Supports both relative and absolute $file path. They differ for PHP-CS-Fixer and PHP_CodeSniffer.
      */
-    private function doesFileMatchPattern(string $filePath, string $ignoredPath): bool
+    public function doesFileMatchPattern(string $filePath, string $ignoredPattern): bool
     {
+        $filePath = PathNormalizer::normalize($filePath);
```

Comment:
revert this, it will cause calles repetitive on loop

Reply:
But I'm not using the loop here anymore. I made the method public and call it from the outside with a single file.

Reply:
The method is called by `doesFileInfoMatchPattern` above. File name should already full path to begin with on caller

Reply:
I have changed it so that the FilePatternSkipper will now receive all string values as an array, so I could use the original FileInfoMatcher's implementation and have reverted it.